tools/versioning.py

```diff
@@ -13,6 +13,7 @@
 VIZ_INIT_FILE = "package/kedro_viz/__init__.py"
 PACKAGE_JSON_FILE = "package.json"
 PACKAGE_JSON_LOCK_FILE = "package-lock.json"
+DEMO_VERSION_FILE = "demo-modular-spaceflights/.version"
```

Comment:
I think this needs to be renamed back to `demo-project` as the rename was reverted.